Antidote Verifier/Sdl.Community.AntidoteVerifier/Sdl.Community.AntidoteVerifier/Utils/CustomTextCollectionVisitor.cs

```diff
@@ -0,0 +1,333 @@
+﻿using Sdl.FileTypeSupport.Framework.BilingualApi;
+using System;
+using System.Collections.ObjectModel;
+using System.Text;
+
+namespace Sdl.Community.AntidoteVerifier
+{
+    public class CustomTextCollectionVisitor : IMarkupDataVisitor
```

Comment:
Can you please explain what was the reason for making this implementation rather than using the existing one? Have you found any issues? If yes can you please let me know.

Also it seems that this implementation is considering tags as part of text. Are tags handled now in Antidote?